include/itkBSplineScatteredDataPointSetToGradientImageFilter.h

```diff
@@ -62,46 +62,45 @@ class BSplineScatteredDataPointSetToGradientImageFilter :
-                 itkGetStaticConstMacro(ImageDimension) >
-  OutputImageType;
-  typedef typename OutputImageType::Pointer OutputImagePointer;
-
-  /** Standard class typedefs. */
-  typedef PointSetToImageFilter< InputPointSetType, OutputImageType > Superclass;
-  typedef SmartPointer< Self >                                        Pointer;
-  typedef SmartPointer< const Self >                                  ConstPointer;
+  /** Standard class type alias. */
+  using Self = BSplineScatteredDataPointSetToGradientImageFilter;
+
+  /** Convenient type alias for simplifying declarations. */
+  using InputPointSetType = TInputPointSet;
+  using InputPointSetPointer = typename InputPointSetType::Pointer;
+  using OutputImageType = Image< CovariantVector< TOutputValueType, itkGetStaticConstMacro(ImageDimension) >,
+                 itkGetStaticConstMacro(ImageDimension) >;
+  using OutputImagePointer = typename OutputImageType::Pointer;
+
+  /** Standard class type alias. */
+  using Superclass = PointSetToImageFilter< InputPointSetType, OutputImageType >;
+  using Pointer = SmartPointer< Self >;
+  using ConstPointer = SmartPointer< const Self >;
 
   /** Method for creation through the object factory. */
   itkNewMacro(Self);
 
   /** Run-time type information (and related methods). */
   itkTypeMacro( BSplineScatteredDataPointSetToGradientImageFilter, PointSetToImageFilter );
 
-  /** Image typedef support. */
-  typedef typename InputPointSetType::PixelType InputPixelType;
+  /** Image type alias support. */
+  using InputPixelType = typename InputPointSetType::PixelType;
   itkStaticConstMacro( InputVectorDimension, unsigned int, InputPixelType::Dimension );
 
-  typedef typename InputPointSetType::PointType                 InputPointType;
-  typedef TOutputValueType                                      OutputValueType;
-  typedef CovariantVector< OutputValueType, ImageDimension >    OutputPixelType;
-  typedef typename OutputImageType::RegionType                  OutputImageRegionType;
-  typedef typename itk::Image< InputPixelType, ImageDimension > InternalImageType;
+  using InputPointType = typename InputPointSetType::PointType;
+  using OutputValueType = TOutputValueType;
+  using OutputPixelType = CovariantVector< OutputValueType, ImageDimension >;
+  using OutputImageRegionType = typename OutputImageType::RegionType;
+  using InternalImageType = typename itk::Image< InputPixelType, ImageDimension >;
 
   /** Internal filter type. */
-  typedef typename itk::BSplineScatteredDataPointSetToImageFilter< InputPointSetType,
-                                                                   InternalImageType > BSplineScatteredDataFilterType;
-  typedef typename BSplineScatteredDataFilterType::Pointer BSplineScatteredDataFilterPointerType;
+  using BSplineScatteredDataFilterType = typename itk::BSplineScatteredDataPointSetToImageFilter< InputPointSetType,
```

Comment:
Maybe break the line at `<`?

Reply:
Still found it too long, although I know this is subjective. Preferred to break at `=` in c577d13.